packages/cli/__tests__/zosfiles/__integration__/create/ds/__snapshots__/cli.files.create.dataset.integration.test.ts.snap

```diff
@@ -199,9 +199,10 @@ exports[`Create  Data Set should display the help 1`] = `
 
       $ zowe zos-files create data-set NEW.DATASET --like EXISTING.DATASET --lrecl 1024
 
-   - Create a data set with type LIBRARY:
+   - Create a data set with default parameters and like flag and
```

Comment:
Thanks for updating the example to a working command 🙂 

I'm not sure if updating the example is the best fix for #1252, or if the command should be updated to support the original example.

IMO it's not intuitive for the `zowe zos-files create data-set` command to fail when the `--like` option isn't provided.

It would be nice if the command `zowe zos-files create data-set NEW.DATASET --data-set-type LIBRARY` worked, using default parameters only.

Thoughts about this @MikeBauerCA @zFernand0 ?

Reply:
Instead of just a reaction I should've used some words 😋 

I do agree that the command should work without the `--like` parameter and get the default parameters based on type of DS being created.